hr_holidays_compute_days/README.rst

```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+===========================
+Employee Compute Leave Days
+===========================
+
+Computes the actual days for which employee will be on leave taking into 
+account both rest days and public holidays
+
+Installation
+============
+
+To install this module, you need to:
+
+* clone the branch 8.0 of the repository https://github.com/OCA/hr
+* add the path to this repository in your configuration (addons-path)
+* update the module list
+* search for "Employee Compute Leave Days" in your addons
+* install the module
+
+Configuration
+=============
+
+Go to your holiday status and specify what leaves should not include rest days
+and holidays in computation
+
+Usage
+=====
+
+
```

Comment:
A scenario using the computation would be nice here.

Please refer to the template to include the button to try your module on Runbot:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst